packages/core/src/routes/session/profile.ts

```diff
@@ -178,4 +181,66 @@ export default function profileRoutes<T extends AnonymousRouter>(router: T, prov
 
     return next();
   });
+
+  router.patch(
+    `${profileRoute}/social`,
+    koaGuard({
+      body: object({
+        connectorId: string(),
+        data: unknown(),
+      }),
+    }),
+    async (ctx, next) => {
+      const { accountId: userId } = await provider.Session.get(ctx);
```

Comment:
Do we need to guard the authorization ts for these social link/unlink actions?

Reply:
Just checked with designer, and you're right. Will update ASAP